lib/standards/html-elms.js

```diff
@@ -0,0 +1,507 @@
+// Source: https://www.w3.org/TR/html-aria/#allowed-aria-roles-states-and-properties
+// Source: https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings
+const htmlElms = {
+	a: {
+		features: [
+			{
+				matches: {
+					attributes: {
+						href: '/.*/'
+					}
+				},
+				allowedRole: true,
+				allowedAttrs: true,
```

Comment:
I think a simpler way to do this is to use "noAriaAttrs", we'd only have to specify the prop on a few roles.